package.json

```
@@ -8,23 +8,18 @@
     "buildsInSource": "_build"
   },
   "dependencies": {
-    "ocaml": "~4.7.0",
-    "revery": "^0.9.0",
-    "@opam/dune": "*"
+    "@esy-ocaml/reason": "^3.6.0",
+    "revery": "revery-ui/revery#431e515a00ee46f7bc614a10c2a219aae13179e0",
+    "ocaml": "4.10.0000"
   },
```

`revery-packager` requires `esy-macdylibbundler` on macOS

```suggestion
    "@esy-ocaml/reason": "^3.6.0",
    "revery": "revery-ui/revery#431e515a00ee46f7bc614a10c2a219aae13179e0",
    "ocaml": "4.10.0000",
    "esy-macdylibbundler": "*"
  },
```